akka-stream/src/main/scala/akka/stream/impl/fusing/Interpreter.scala

```diff
@@ -9,6 +9,7 @@ import scala.util.control.NonFatal
 import akka.stream.stage._
 import akka.stream.Supervision
 import akka.stream.impl.ReactiveStreamsCompliance
+import akka.stream.FlowMaterializer
```

Comment:
Interpreter does not have type parameters. I like the fact that the interpreter can be ripped out and used with minimal dependencies. I just try to maintain this "simple core" design.